src/content/blog/kai-deep-dive-2024/index.mdx

```diff
@@ -0,0 +1,171 @@
+
+We use the analysis information for:
+
+- Identifying what areas of code need to be updated to move to a new technology
+- Guidance to inform a developer what the issue is and hints of how they may resolve the problem
+
+For example we can look at the below snippet to see an [example of the YAML data](https://github.com/jwmatthews/sample_kantra_reports/blob/main/coolstore-quarkus/out/output.yaml) from a specific analysis issue informing the developer about a concern when moving from JMS to Quarkus' reactive messaging:
+
+````yaml
+incidents:
+  - uri: file:///src/main/....../service/InventoryNotificationMDB.java
+    message: "JMS `Topic`s should be replaced with Micrometer `Emitter`s feeding a Channel. See the following example of migrating\n a Topic to an Emitter:\n \n Before:\n ```\n @Resource(lookup = \"java:/topic/HELLOWORLDMDBTopic\")\n private Topic topic;\n ```\n \n After:\n ```\n @Inject\n @Channel(\"HELLOWORLDMDBTopic\")\n Emitter<String> topicEmitter;\n ```"
+    lineNumber: 60
+````
+
+Additionally, this analysis information is available in a more human friendly [WebUI view](https://jwmatthews.github.io/sample_kantra_reports/coolstore-quarkus/out/static-report/#/applications)
+![Analysis Report](AnalysisReport_WebUI.png)
+
+#### Solved Examples
+
+##### What do we mean by 'Solved Examples'
+
+Application Modernization engagements inside a large organization typically encompass ~100s of applications that need to be migrated to a new technology. Often these applications share a large number of similar issues. As an organization successfully migrates a handful of applications they begin to encounter repeated patterns of issues they have already solved. One of the big goals with Kai is helping an organization tap into these prior solutions and leverage them for new migration needs, hence what we refer to as a 'Solved Example'.
+
+##### How do we determine if we have a 'Solved Example' for a given Analysis Incident
+
+Kai will help address similar migration problems by using the data inside of Konveyor which has an organization wide view of the entire application portfolio.
+![Information in Konveyor](Information_In_Konveyor.png)
+
+Kai is able to find occurrences of when an application previously had the same problem and then was successfully migrated.
+
+![Solved Examples](Kai_SimilarSolvedExamples.png)
+
+##### Use of 'Solved Examples' in a Few Shot Prompt
+
+Once Kai has found how a similar problem has been solved elsewhere in the organization it extracts part of that information and gives it to the LLM to help consider as a ['few shot'](https://www.promptingguide.ai/techniques/fewshot) example.
+
+![Few Shot Prompt](Prompt_Example_small.png)
+
+### LLM Specific Concerns
```

Comment:
It feels like this is more laying out the set of problems we need to tackle in the future, I don't feel like this header accurately describes the section